### PR TITLE
B002 - Create Database script

### PR DESCRIPTION
**Pull request by adammarshall745**
This script is to create the database.

### CODE REVIEW

**Review by BlankAmber — APPROVED** [review]
FANTASTIC SCRIPT!

**Comment by BlankAmber** [comment]
The permissions script needs to be made, I am unable to test this. Please do not commit the permissions script

File: database/DatabaseScript.sql
Comment (jkelso662) [inline database/DatabaseScript.sql:1]: Will this script not fail if the schema is already created?

**Review by jkelso662 — CHANGES_REQUESTED** [review]
There's a lot in here that is not to do with your schema script, and also stuff that I think is already in your branch sign-off branch. Worth pulling the latest changes to make sure. Checkout a couple of other comments.

**Comment by lullabelle** [comment]
Where is the rest of the database script? There are only 2 lines in here. This needs to be executable should the database need recreated.
Gitignore and other bits and pieces in here not related to ticket

File: database/DatabaseScript.sql
Comment (BlankAmber) [reply database/DatabaseScript.sql:1]: Fixed in [#bb90436 ](https://github.com/BlankAmber/Kainos-RRAS-API/pull/2/commits/bb90436e4942bdfbbf1f7439a218bff3f602c78f)

**Comment by BlankAmber** [comment]
> There's a lot in here that is not to do with your schema script, and also stuff that I think is already in your branch sign-off branch.
> Gitignore and other bits and pieces in here not related to ticket
This should be fixed now

**Comment by BlankAmber** [comment]
> Where is the rest of the database script? There are only 2 lines in here.

We were planning on adding the tables incrementally as we do the tickets. E.g. US001 would add the `job_roles` table, ES040 would add the `users`, `tokens`, and `roles` tables. Would that be okay, or should we add more tables to start with?

**Review by jkelso662 — APPROVED** [review]

**Pull request merged**